### PR TITLE
Fix typos in guides

### PR DESCRIPTION
**Pull request by ArmaanMcleod**
Fixed typos in multiple guide step files.

### CODE REVIEW

**Review by brk3 — COMMENTED** [review]
Looks good to me, however, there's at least one other typo in the guide (see step 1. on https://github.com/mspnp/aks-secure-baseline/blob/main/02-ca-certificates.md)

Might be nice to run all steps through a spellchecker and fix all in one go?

**Comment by ArmaanMcleod** [comment]
@brk3 Did another run through, made some more changes. Let me know what you think 🙂

**Review by brk3 — APPROVED** [review]
Thanks @ArmaanMcleod 👍

File: 07-workload-prerequisites.md [hunk 1]
```diff
@@ -69,4 +69,4 @@ The AKS Cluster has been enrolled in [GitOps management](./06-gitops.md), wrappi
 
 ### Next step
 
-:arrow_forward: [Configure AKS Ingress Controller with Azure Key Vault integration](./08-secret-managment-and-ingress-controller.md)
+:arrow_forward: [Configure AKS Ingress Controller with Azure Key Vault integration](./08-secret-management-and-ingress-controller.md)
```
Comment (ckittel) [inline 07-workload-prerequisites.md:72]: Just a heads up,  `09-workload.md` also has this URL in it, can you update it there as well?  After that, I can merge this in.

**Review by ckittel — CHANGES_REQUESTED** [review]

File: 07-workload-prerequisites.md (hunk 1 above)
Comment (ArmaanMcleod) [reply 07-workload-prerequisites.md:72]: @ckittel Ah good spot, fixed 🙂 .

**Review by ckittel — APPROVED** [review]
w00t! 🚀 

Thanks so much.

**Pull request merged**